src/org/loklak/harvester/TwitterScraper.java

```diff
@@ -1,858 +1,863 @@
+                // read until closing p tag to account for new lines in tweets
+                while (input.lastIndexOf("</p>") == -1){
+                    input = input + ' ' + br.readLine();
+                }
+                prop tweettext = new prop(input, p, null);
+                props.put("tweettext", tweettext);
+                continue;
+            }
+            if ((p = input.indexOf("class=\"ProfileTweet-actionCount")) > 0) {
+                if (parsing_retweet) {
+                    prop tweetretweetcount = new prop(input, p, "data-tweet-stat-count");
+                    props.put("tweetretweetcount", tweetretweetcount);
+                    parsing_retweet = false;
+                }
+                if (parsing_favourite) {
+                    props.put("tweetfavouritecount", new prop(input, p, "data-tweet-stat-count"));
+                    parsing_favourite = false;
+                }
+                continue;
+            }
+            // get images
+            if ((p = input.indexOf("data-image-url=")) >= 0) {
+                String image_url = new prop(input, p, "data-image-url").value;
+                if (!image_url.endsWith(".jpg") && !image_url.endsWith(".png")) {
+                    DAO.log("strange image url: " + image_url);
+                }
+                images.add(image_url);
+                continue;
+            }
+            // get images
+            if ((p = input.lastIndexOf("background-image:url('")) >= 0) {
+                int q = input.lastIndexOf("'");
+                if (q > p + 22) {
+                    String image_url = input.substring(p + 22, q);
+                    if (!image_url.endsWith(".jpg") && !image_url.endsWith(".png")) {
+                        DAO.log("strange image url: " + image_url);
+                    }
+                    images.add(image_url);
+                }
+                continue;
+            }
+            // we have two opportunities to get video thumbnails == more images; images in the presence of video content should be treated as thumbnail for the video
+            if ((p = input.indexOf("class=\"animated-gif-thumbnail\"")) > 0) {
+                String image_url = new prop(input, 0, "src").value;
+                images.add(image_url);
+                continue;
+            }
+            if ((p = input.indexOf("class=\"animated-gif\"")) > 0) {
+                String image_url = new prop(input, p, "poster").value;
+                images.add(image_url);
+                continue;
+            }
+            if ((p = input.indexOf("<source video-src")) >= 0 && input.indexOf("type=\"video/") > p) {
+                String video_url = new prop(input, p, "video-src").value;
+                videos.add(video_url);
+                continue;
+            }
+            if (input.indexOf("AdaptiveMedia-videoContainer") > 0) {
+                /* String tweetUrl = props.get("tweetstatusurl").value;
```

Comment:
Only this if loop's body is changed. I've commented out the process for getting the video URLs while commenting.